eve/meta/start/licenses/mit.txt

```diff
@@ -0,0 +1,22 @@
+                        MIT License
```

Comment:
Do you think there is a way we can download these dynamically? - I'm thinking if they ever change we've them hard coded. I imagine there might be a URL we can hit that will pull these down.

What do you think?